pymc_experimental/gp/latent_approx.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+import aesara.tensor as at
+import pymc as pm
+
+from pymc.gp.util import (
+    JITTER_DEFAULT,
+    cholesky,
+    conditioned_vars,
+    infer_size,
+    replace_with_values,
+    solve_lower,
+    solve_upper,
+    stabilize,
+)
+
+
+class LatentApprox(pm.gp.Latent):
+    ## TODO: use strings to select approximation, like pm.gp.MarginalApprox?
+    pass
+
+
+class ProjectedProcess(pm.gp.Latent):
+    ## AKA: DTC
+    def __init__(self, n_inducing, *, mean_func=pm.gp.mean.Zero(), cov_func=pm.gp.cov.Constant(0.0)):
```

Comment:
I would add type hints to public-facing API and at least simple docstrings. Can be done in a separate PR if you prefer.